### DISCUSSION TITLE
Tutorials

### DISCUSSION THREAD

**Discussion by agataszade** [ImmuneDynamics/Spectre#187] [Q&A] [answered]
Hi,
I'm new to spectre and wanted to start using it. The problem is I can't open any tutorials which are on https://wiki.centenary.org.au . Are they available anywhere else?

**Comment by SamGG** [comment]
Hi,
In https://github.com/ImmuneDynamics/Spectre/workflows, there are many. Classic workflows is probably what you are looking for, I guess.

**Reply by agataszade** [reply]
Thank you! this is helpful, but first I'm looking for some description (like here for CytoTree https://ytdai.github.io/CytoTree/index.html); I think the tutorials were available, as there are many links here https://immunedynamics.io/spectre/cytometry/, but the pages just don't load

**Reply by SamGG** [reply]
@tomashhurst  It seems that wiki.centenary.org.au is off?

**Reply by ghar1821** [reply]
@agataszade I just tried accessing the immunedynamics website and I can load the pages. Can you try again?

@SamGG The uni might have taken the website off for maintenance. I've seen them doing this a few times in the past. We'll look into migrating some stuff out of the centenary website into the immunedynamics website.

**Reply by SamGG** [reply]
Centenary still leads to a timeout from France.

**Reply by agataszade** [reply]
@ghar1821 the immunodynamics website is ok, but the pages with actual tutorials ex. "Initial data preparation" https://wiki.centenary.org.au/x/1eRfCQ don't open

**Comment by tomashhurst** [comment]
Hi all,

Sorry for the inconvenience! The hosting setup for the ‘centenary’ pages has changed — we’ll move the contents to the main site shortly to avoid any access issues.

Tom

________________________________

**Reply by agataszade** [reply]
Thank you!

**Comment by tomashhurst** [comment, marked as answer]
Hi all,

We have updated most of the links on the website for those offline pages.
They are in a PDF format until we can get everything moved properly.

All the PDFs can be found here: Spectre PDFs
<https://unisyd-my.sharepoint.com/:f:/g/personal/thomas_ashhurst_sydney_edu_au/EhvUxKBTE7FCqBfNkZvtIpoBb-mnoFYhuOf-wKhQrfsRnw>
.

Tom

**Reply by mollieb12** [reply]
Hi Tom!

It looks like that link has expired - is there still a way to access these tutorials?

Thanks!
Mollie
